negotiation/diff.emu

```diff
@@ -333,15 +333,21 @@
         The abstract operation GetStringOrBooleanOption extracts the value of the property named _property_ from the provided _options_ object. It returns either _trueValue_, _falsyValue_, _fallback_, or one of the elements of the List _values_. The following steps are taken:
       </p>
 
-      <emu-alg>
+     <emu-alg>
         1. Let _value_ be ? Get(_options_, _property_).
-        1. If _value_ is *undefined*, return _fallback_.
-        1. If _value_ is *true*, return _trueValue_.
-        1. Let _valueBoolean_ be ToBoolean(_value_).
-        1. If _valueBoolean_ is *false*, return _falsyValue_.
-        1. Let _value_ be ? ToString(_value_).
-        1. If _values_ does not contain an element equal to _value_, throw a *RangeError* exception.
-        1. Return _value_.
+        1. If _value_ is not *undefined*, then
+          1. Assert: _type_ is *"boolean"* or *"string"*.
+          1. If _type_ is *"boolean"*, then
+            1. Let _value_ be ToBoolean(_value_)
+            1. If _value_ is *true*, return _trueValue_.
+            1. If _value_ is *false*, return _falsyValue_.
```

Comment:
This section doesn't make sense to me.

- `_type_` is not defined anywhere I can see
- If `_type_` is already a boolean, then why do we need `ToBoolean`?